hectorsector-orlando.md

```diff
@@ -0,0 +1,10 @@
+#  Orlando, FL
+
```

Comment:
@githubteacher Good, but which part of Orlando? That would give me a better idea when I visit you! 🥇